lib/scripts/repo_export_in_batches.rb

```diff
@@ -40,7 +40,8 @@
     )
     puts "---------\n#{converter[:model_class].name} has id: #{record_for_export.id}\n---------"
 
-    if converted_record.present?
+    converted_record_json = JSON.parse(converted_record.to_json)
+    if converted_record_json.present?
       file_path = "tmp/export/#{converter[:model_class].name.underscore}_#{record_for_export.id}.json"
       File.write(file_path, converted_record.to_json(pretty: true))
```

Comment:
If sticking with the `converted_record_json` route, can we reuse it here, instead of calling `to_json` again? Maybe add the `(pretty: true)` to the variable?

Reply:
Can we use the new `converted_record_json` here?

Reply:
Ah, missed this. I'll update it and push. Thanks for double checking me.